tests/lib/snaps/test-snapd-with-configure/service

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo "service running"
+sleep infinity
```

Comment:
Please exec sleep instead.

```suggestion
exec sleep infinity
```

Reply:
Out of curiosity, why exec?

Reply:
Without exec the primary process is the bash process, with exec the primary process is sleep.
There's less memory used, less bash running around and you can actually do stuff with sleep directly using service APIs.